### ISSUE TITLE
Windows support

### ISSUE THREAD

**Issue by thegtproject** [golangci/golangci-lint#14]
Console painting issues for windows, e.g.

![image](https://user-images.githubusercontent.com/28569582/40577748-cf64bea0-60c7-11e8-9eb2-1e990dc1c0d0.png)

**Comment by syncore** [comment]
![color](https://user-images.githubusercontent.com/1934331/40578732-3dfa6e68-60e7-11e8-8659-0b820c64aa54.png)

related, perhaps:

https://github.com/fatih/color/issues/91
https://github.com/fatih/color/pull/87

**Comment by golangci** [comment]
@thegtproject @syncore thank you for reporting the issue!
while we're preparing the fix you can use option `--out-format` to output issues without colors:
```
golangci-lint run --out-format=line-number
```
but it doesn't work with `golangci-lint linters`

**Comment by syncore** [comment]
Thank you and thank you for this useful tool! I was doing some testing locally, and managed to get it to work following issue #91 in the color library linked above. I am not sure if it breaks
on non-Windows so I have kept it local for now, but I will follow your recommendation to use the non-colored out format.

For reference, the modifications I made (there might be better ways to do this, but I was curious to see if I could get it to work at all):

changed  fmt.Printf in the printLinterConfigs function at golangci-lint/blob/master/pkg/commands/linters.go:24

to:

`fmt.Fprintf(color.Output, "%s: %s [fast: %t]\n", color.YellowString(lc.Linter.Name()),
			lc.Linter.Desc(), !lc.DoesFullImport)`

and doing the same in the executeLinters function at golangci-lint/blob/master/pkg/commands/linters.go:51

`fmt.Fprintf(color.Output, "%s: %s\n", color.YellowString(p), strings.Join(linterNames, ", "))`

as well as similar changes in golangci\golangci-lint\pkg\printers\text.go:

`fmt.Fprintln(stdOut, outStr)` to: `fmt.Fprintln(color.Output, outStr)`

`fmt.Fprintf(stdOut, "%s: %s\n", pos, text)` to: `fmt.Fprintf(color.Output, "%s: %s\n", pos, text)`

`fmt.Fprintf(stdOut, "%s%s\n", prefix, p.SprintfColored(color.FgYellow, "^"))` to: `fmt.Fprintf(color.Output, "%s%s\n", prefix, p.SprintfColored(color.FgYellow, "^"))`

![c1](https://user-images.githubusercontent.com/1934331/40579283-e42ac4ae-60f1-11e8-8afb-65bf48e28bde.png)

![c2](https://user-images.githubusercontent.com/1934331/40579285-e75c3662-60f1-11e8-9816-955c8cf3c7cd.png)
